decidim-core/app/commands/decidim/create_omniauth_registration.rb

```diff
@@ -73,10 +75,11 @@ def create_or_find_user
           file = url.open
           @user.avatar.attach(io: file, filename:)
         end
+        @user.tos_agreement = form.tos_agreement
```

Comment:
I noticed that when i am registering with omniauth, even though i check the "Tos", i am being redirected to the "Terms of service" page. On a closer look, we need to add a value for `accepted_tos_version` in order to avoid being redirected to "Tos page". 

```suggestion
        @user.tos_agreement = form.tos_agreement
        @user.accepted_tos_version = Time.current 
```